test/unit/components/VList/VListGroup.spec.js

```diff
@@ -153,7 +153,23 @@ test('VListGroup.js', ({ mount }) => {
     await wrapper.vm.$nextTick()
     expect(unregister).toBeCalledWith(wrapper.vm._uid)
   })
+  it('should toggle when it is focused and enter key is pressed', async () => {
+    const wrapper = mount(VListGroup, {
+      attachToDocument: true,
+      provide: {
+        listClick: () => {}
+      }
+    })
+    
+    const input = jest.fn()
+    wrapper.vm.$on('input', input)
+    wrapper.vm.onKeyDown({ keyCode: 13, stopPropagation: jest.fn() })
```

Comment:
Trigger an actual event instead of calling the method directly.

Reply:
done